chess-AI/.gitignore

```diff
@@ -5,8 +5,13 @@
 # Our first model!
 !model.pt
 
+# BayesianElo program used for computing elo of agents
+BayesianElo/
```

Comment:
Where do we get this program?

Reply:
https://github.com/ddugovic/BayesianElo

Reply:
This should probably be included in a the ReadMe for chess-AI. I am going to work on creating a more detailed readme for chess-AI because we have nothing explaining figshare, or params.json.